app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NOTIFICATION_POLICY" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.TETHER_PRIVILEGED"/>
```

Comment:
IIRC, this doesn't seem to be documented. Does it not require the current app (i.e. Easer) to be system app?

Reply:
I think this permission needs switch on advanced permission 'Apps that can change system settings' in 'APPS SETTINGS'. Link for this https://www.youtube.com/watch?v=-GTMN0-VTrE